packages/rocketchat-api/server/default/info.js

```diff
@@ -1,5 +1,13 @@
 RocketChat.API.default.addRoute('info', { authRequired: false }, {
 	get: function() {
-		return RocketChat.Info;
+		if (this.request.headers['x-user-id'] != null && RocketChat.authz.hasRole(this.request.headers['x-user-id'], 'admin')) {
```

Comment:
This would allow anyone who can get ahold of an admin's user id (which isn't trivial) to be able to pass in the `x-user-id` as a header and get this information, which means this isn't protecting any data at all. There's a few ways you can do this, although one way sticks out which is the better and cleaner option of them which is to create a new [helper method](https://github.com/RocketChat/Rocket.Chat/tree/develop/packages/rocketchat-api/server/v1/helpers) that is something like `this.getLoggedInUser()` and it returns the user object only if the user has passed in valid items.